editor/editor_audio_buses.cpp

```diff
@@ -238,7 +238,14 @@ void EditorAudioBus::update_bus() {
 		fx->set_cell_mode(0, TreeItem::CELL_MODE_CHECK);
 		fx->set_editable(0, true);
 		fx->set_checked(0, AudioServer::get_singleton()->is_bus_effect_enabled(index, i));
-		fx->set_text(0, afx->get_name());
+		if (afx->get_name().is_empty()) {
+			fx->set_text(0, afx->get_path().get_file());
+		} else {
+			fx->set_text(0, afx->get_name());
+		}
+		if (!afx->is_connected("changed", callable_mp(this, &EditorAudioBus::update_bus))) {
+			afx->connect("changed", callable_mp(this, &EditorAudioBus::update_bus));
```

Comment:
I think it's ok, renaming does not happen that often.

But instead of string for the signal name, you could use `SNAME` or `changed` member from `scene_string_names.cpp`.

Reply:
Done!